stock_warehouse_flow/models/stock_location_route.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import api, fields, models
+
+
+class StockLocationRoute(models.Model):
+    _inherit = "stock.location.route"
+
+    flow_id = fields.Many2one(
+        comodel_name="stock.warehouse.flow",
+        ondelete="set null",
+        string="Routing Flow",
+    )
+    applicable_flow_ids = fields.One2many(
```

Comment:
What is/was the purpose of this?
The flow defines new picking types for the route's rules, so the search does not make sense to me

Reply:
This is to inform the user that one route could be replaced by another flow (if the criterias are matching), it's not used for other purpose.

Reply:
got it :)